app/services/member-consequences.js

```diff
@@ -1,6 +1,68 @@
 import Ember from 'ember';
 
 export default Ember.Service.extend({
+  store: Ember.inject.service(),
+  sessionStorage: Ember.inject.service('session'),
+  calculateBmi(member, chapter, tags) {
+
+    // Get BMI option, if it exists
+    //@TODO: Questions need a separate boolean property for 'hidden' so that hidden questions can also be referenced by their 'type'
+    var bmi_option = this.get('store').peekAll('option').filterBy('text', '__input-bmi').objectAt(0);
+
+    // Proceed to attempt to calculate BMI only if the BMI option exists
+    if (bmi_option) {
+      let bmi_question = bmi_option.get('question'),
+        bmi_tag = this.get('store').peekAll('tag').filterBy('questionId', parseInt(bmi_question.get('id'))).objectAt(0),
```

Comment:
Use the `tags` argument passed into this method. Or don't pass the tags into this method.